ext/nnstreamer/tensor_decoder/tensordec-pose.c

```diff
@@ -0,0 +1,526 @@
+/**
+ * GStreamer / NNStreamer tensor_decoder subplugin, "Pose estimation"
+ * Copyright (C) 2018 Samsung Electronics Co. Ltd.
+ * Copyright (C) 2018 Jijoong Moon <jijoong.moon@samsung.com>
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Library General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Library General Public License for more details.
+ *
+ */
+/**
+ * @file        tensordec-pose.c
+ * @date        13 May 2019
+ * @brief       NNStreamer tensor-decoder subplugin, "pose estimation",
+ *              which converts tensors to video stream w/ pose on
+ *              transparent background.
+ *              This code is NYI/WIP and not compilable.
+ *
+ * @see         https://github.com/nnsuite/nnstreamer
+ * @author      Jijoong Moon <jijoong.moon@samsung.com>
+ * @bug         No known bugs except for NYI items
+ *
+ * option1: Video Output Dimension (WIDTH:HEIGHT)
+ * option2: Input Dimension (WIDTH:HEIGHT)
+ */
+
+#ifndef _GNU_SOURCE
```

Comment:
plz remove _GNU_SOURCE, it is unnecessary.

```
#ifndef _GNU_SOURCE
#define _GNU_SOURCE
#endif
```

Reply:
Done.